config.js

```diff
@@ -66,8 +66,8 @@ const ConfigOptions = () => {
         img: {
             imgDir: 'img',
             svgSpriteFilename: 'sprite.svg',
-            optimiseImages: false,
-            spriteSvgs: false
+            optimiseImages: true,
```

Comment:
Do you think this is worth a major version bump potentially? On the fence myself – I don't think it would cause any issues but just wanted to double check

Reply:
Maybe just leave it as a minor then – I don't think it will. It may just optimise some images for them…